common/djangoapps/third_party_auth/models.py

```diff
@@ -140,6 +140,14 @@ class ProviderConfig(ConfigurationModel):
             "authentication using the correct link is still possible."
         ),
     )
+    drop_existing_session = models.BooleanField(
+        default=False,
+        help_text=_(
+            "Whether to drop an existing session when accessing a view decorated with "
+            "third_party_auth.decorators.tpa_hint_ends_existing_session when a tpa_hint "
+            "URL query parameter mapping to this provider is included in the request."
```

Comment:
This is fine, but still will make little sense to your average Open edX administrator. I would prefer "When this option is selected, users who follow hinted links from an external site to this site will be forced to login again using SSO, even if they were already logged in. This is useful if people may be using shared computers for SSO logins. (Specifically, this will drop any existing session when a tpa_hint URL query parameter mapping to this provider is included in the request, and the user is accessing one of the URLs/views view decorated with tpa_hint_ends_existing_session.)"